CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   permits callback using code to be written without indentation.
 - Fixed a bug where the warning for `todo` would not print the type of the code
   to complete.
+- Fixed float-size in bitstring
```

Comment:
Could you explain what this means? Ideally a user can read this and know what new things they can do.

Reply:
Could you update this please? 🙏